### PR TITLE
Feature/ui 5212 add code section for ui kit demo layout page

### PR DESCRIPTION
**Pull request by EduardStransky**

### CODE REVIEW

File: data/styles/base/custom/custom-presentation.less [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    background: @selected-tab-color;
+    border:0;
+  }
+  pre[class*="language-"] {
+    background: @selected-tab-color;
+    border:0;
+  }
+}
+.code-container {
+  margin-left:-15px;
```
Comment (valentinbica) [inline data/styles/base/custom/custom-presentation.less]: Try combining rules

File: demo/layout.php [hunk 2]
```diff
@@ -5,24 +5,26 @@
...
+
         <!-- PAGE:Start -->
             <div class="main-container" id="main-container">
                 <div class="main-container-inner">
                 <!-- SIDEBAR:Start -->
                 <?php include_once "modules/_mod_sidebar.php"?>
                 <!-- SIDEBAR:End -->
                 <!-- CONTENT:Start -->
-                <div class="main-content">
+                <div id="cucu" class="main-content">
```
Comment (georgevlada) [inline demo/layout.php]: Please change this id name.

File: data/styles/base/custom/custom-utilities.less [hunk 3]
```diff
@@ -576,6 +611,22 @@
...
 
+.no-padding {
+  padding: 0;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.no-border-top{
```
Comment (valentinbica) [inline data/styles/base/custom/custom-utilities.less]: Use space before {

File: data/styles/base/custom/custom-utilities.less [hunk 4]
```diff
@@ -576,6 +611,22 @@
...
+  border-top: 0;
+}
+
+.no-border-bottom{
```
Comment (valentinbica) [inline data/styles/base/custom/custom-utilities.less]: Use space before {

File: dist/js/demo_helpers.js [hunk 5]
```diff
@@ -15,27 +15,93 @@ function escapeHtml(unsafe) {
             .replace(/'/g, "&#039;");
 };
 
-function cutWhiteSpace() {
-    var offset = escapeHtml(example).match(/^\s+/)[0].length;
-    code = escapeHtml(example).split('\n').map(function (line) {
+function typeOfTag($obj) {
+    if ($obj.is('script')) {
```
Comment (valentinbica) [inline dist/js/demo_helpers.js]: Use switch
And
if(!$obj.attr('src'))

File: dist/js/demo_helpers.js [hunk 6]
```diff
@@ -15,27 +15,93 @@ function escapeHtml(unsafe) {
...
+    if ($obj.is('link')) {
+        return 'style-external';
+    }
+}
+
+function getFormattedCode(htmlContent) {
+
+    var codeLines = escapeHtml(htmlContent).split('\n');
+
+    if (codeLines[0].slice(offset - 1) == '') {
```
Comment (valentinbica) [inline dist/js/demo_helpers.js]: ===

File: dist/js/demo_helpers.js [hunk 7]
```diff
@@ -15,27 +15,93 @@ function escapeHtml(unsafe) {
...
-    $('.show-panel-example').each(function () {
-        var example = $(this).find('[data-showcase="example"]').html();        
-        if(example){
-            var offset = escapeHtml(example).match(/^\s+/)[0].length;
-            code = escapeHtml(example).split('\n').map(function (line) {
-                return line.slice(offset - 1);
-            });
-            $(this).find('[data-showcase="code"]').append(code.join('\n'));
+function getFormattedDependencies(dependencies) {
+    var dependenciesArr = dependencies.split(",");
```
Comment (valentinbica) [inline dist/js/demo_helpers.js]: Single quotes please

File: dist/js/demo_helpers.js [hunk 8]
```diff
@@ -15,27 +15,93 @@ function escapeHtml(unsafe) {
...
+        var example = $this.html();
+
+        if (example) {
+            var formattedHtml = getFormattedCode(example);
+            $this.parent().find('.html-source code').append(formattedHtml);
+
+            var dependencies = $this.attr('data-dependencies');
+            if (dependencies) {
+                $(getFormattedDependencies(dependencies)).each(function () {
+                    var $dependency = $(this);
```
Comment (valentinbica) [inline dist/js/demo_helpers.js]: Pass $(this) directly to function

File: dist/js/demo_helpers.js [hunk 9]
```diff
@@ -55,8 +121,28 @@ function updateSideBarCode() {
             break;
     }
 
-    $('#sidebar_classes').html(sidebarClasses);
-    $('#sidebar_control_button_icon_classes').html(controlButtonIconClasses);
+    var $sidebarClasses = $('#sidebar_code code > span').find('.attr-value').eq(1);
+    var $sidebarToggleButtonIcon = $('#sidebar_code code > span').eq(60).find('.attr-value').eq(0);
```
Comment (valentinbica) [inline dist/js/demo_helpers.js]: What does 60 mean ?

File: dist/js/demo_helpers.js [hunk 10]
```diff
@@ -15,27 +15,93 @@ function escapeHtml(unsafe) {
             .replace(/'/g, "&#039;");
 };
 
-function cutWhiteSpace() {
-    var offset = escapeHtml(example).match(/^\s+/)[0].length;
-    code = escapeHtml(example).split('\n').map(function (line) {
+function typeOfTag($obj) {
```
Comment (georgevlada) [inline dist/js/demo_helpers.js]: I think the corect name for this function is getTypeOfCode.

**Pull request merged**